model/team.go

```diff
@@ -19,7 +19,7 @@ type Team struct {
 	UpdateAt       int64  `json:"update_at"`
 	DeleteAt       int64  `json:"delete_at"`
 	Name           string `json:"name"`
-	Domain         string `json:"domain"`
+	URLId          string `json:"urlid"`
```

Comment:
The name comes from the fact that it is not the team name. It is a separate field that might have nothing to do with the team name. It is user configurable. So someone could call their team "My Company" and have their urlid be mycompany or mc or a38985. 

I certainty agree with you with the consistency. It annoys me. 0/5 if you want me to fix it before merge.